guide/README.md

```diff
@@ -497,6 +497,7 @@ func semrelease(lock):
 
 The mutex profiler has limitations similar to the block profiler:
 
+- ⚠️ The contention counts and delay times in a mutex profile are adjusted at reporting time based on the *most recent* configured sampling rate, rather than at sample time. As a result, programs which change the mutex profile fraction in the middle of execution can see skewed counts and delays.
```

Comment:
NIT: This strikes me as the lowest impact issue with the mutex profiler, so I'd move it to the bottom of the list?

Reply:
Works for me, I made the change.